package.json

```diff
@@ -12,7 +12,7 @@
     "@react-native-community/push-notification-ios": "^1.8.0",
     "appcenter": "4.2.0",
     "appcenter-crashes": "4.2.0",
-    "bluedot-react-native": "2.0.5",
+    "bluedot-react-native": "git://github.com/Bluedot-Innovation/Bluedot-React-Native-Plugin.git",
```

Comment:
Any reason to merge this before we can use the published dependency?

Reply:
Once the bluedot plugin is published after QA basic sanity, this will be updated with the published version and PR will be merged afterwards.

Reply:
Yeah, perfect 👍